android/build.gradle

```diff
@@ -38,10 +38,10 @@ android {
 dependencies {
     //noinspection GradleDynamicVersion
     implementation "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
-    implementation 'com.yoti.mobile.android.sdk:yoti-sdk-doc-scan:2.9.3'
-    implementation 'com.yoti.mobile.android.sdk:yoti-sdk-doc-scan-sup:2.9.3'
-    implementation 'com.yoti.mobile.android.sdk:yoti-sdk-liveness-zoom:2.9.3'
-    implementation 'com.yoti.mobile.android.sdk:yoti-sdk-facecapture:2.9.3'
+    implementation 'com.yoti.mobile.android.sdk:yoti-sdk-doc-scan:3.0.0'
+    implementation 'com.yoti.mobile.android.sdk:yoti-sdk-doc-scan-sup:3.0.0'
+    implementation 'com.yoti.mobile.android.sdk:yoti-sdk-liveness-zoom:3.0.0'
+    implementation 'com.yoti.mobile.android.sdk:yoti-sdk-facecapture:3.0.0'
```

Comment:
Can you please add the references (dependency + comments) to the `unbundled` version of the `facecapture` module? Same as [here](https://github.com/getyoti/yoti-doc-scan-android/pull/40/files#diff-51a0b488f963eb0be6c6599bf5df497313877cf5bdff3950807373912ac1cdc9).